src/contracts/implementations/arbitrable/MultipleArbitrableTransaction.js

```diff
@@ -0,0 +1,333 @@
+   * @param {bytes} arbitratorExtraData Extra data for the arbitrator. (default empty string)
+   * @param {string} metaEvidenceUri Uri meta-evidence. (default empty string)
+   * @returns {object} truffle-contract Object | err The deployed contract or an error
+   */
+  createArbitrableTransaction = async (
+    account = this._Web3Wrapper.getAccount(0),
+    arbitratorAddress,
+    seller,
+    value,
+    timeout = 3600,
+    arbitratorExtraData = 0x0,
+    metaEvidenceUri
+  ) => {
+    await this.loadContract()
+
+    try {
+      return this.contractInstance.createTransaction(
+        arbitratorAddress,
+        timeout,
+        seller,
+        arbitratorExtraData,
+        metaEvidenceUri,
+        {
+          from: account,
+          value: this._Web3Wrapper.toWei(value, 'ether'),
+          gas: 800000 // FIXME gas hardcoded maybe use estimateGas before
+        }
+      )
+    } catch (err) {
+      console.error(err)
+      throw new Error(errorConstants.UNABLE_TO_CREATE_TRANSACTION)
+    }
+  }
+
+  /**
+   * Pay the party B. To be called when the good is delivered or the service rendered.
+   * @param {string} account - Ethereum account (default account[0]).
+   * @param {number} transactionId - The index of the transaction.
+   * @param {amount} amount - Part or all of the amount of the good or the service.
+   * @returns {object} - The result transaction object.
+   */
+  pay = async (
+    account = this._Web3Wrapper.getAccount(0),
+    transactionId,
+    amount
+  ) => {
+    await this.loadContract()
+
+    try {
+      return this.contractInstance.pay(transactionId, amount, {
+        from: account,
+        value: 0
+      })
+    } catch (err) {
+      console.error(err)
+      throw new Error(errorConstants.UNABLE_TO_PAY_SELLER)
+    }
+  }
+
+  /**
+   * Reimburse the party A. To be called when the good is not delivered or the service rendered.
+   * @param {string} account - Ethereum account (default account[0]).
+   * @param {number} transactionId - The index of the transaction.
+   * @param {amount} amount - Part or all of the amount of the good or the service.
+   * @returns {object} - The result transaction object.
+   */
+  reimburse = async (
+    account = this._Web3Wrapper.getAccount(0),
+    transactionId,
+    amount
+  ) => {
+    await this.loadContract()
+
+    try {
+      return this.contractInstance.reimburse(transactionId, amount, {
+        from: account,
+        value: 0
+      })
+    } catch (err) {
+      console.error(err)
+      throw new Error(errorConstants.UNABLE_TO_REIMBURSE_BUYER)
+    }
+  }
+
+  /**
+   * Pay the arbitration fee to raise a dispute. To be called by the party A.
+   * @param {string} account - Ethereum account (default account[0]).
+   * @param {number} transactionId - The index of the transaction.
+   * @param {number} arbitrationCost - Arbitration cost.
+   * @returns {object} - The result transaction object.
+   */
+  payArbitrationFeeByBuyer = async (
```

Comment:
Buyer/Seller seems a little use case specific. What about something like Sender/Recipient?

Reply:
buyer/seller sounds generic for me and it's this wording buyer/seller is used in the smart contract

Reply:
Ah ok. To me it sounds like it has to be for a transaction of goods. But if its in the smart contract it is fine